Sources/NIOSSL/NIOSSLHandler.swift

```diff
@@ -546,6 +546,28 @@ public class NIOSSLHandler : ChannelInboundHandler, ChannelOutboundHandler, Remo
     }
 }
 
+extension NIOSSLHandler {
+    /// Variable that can be queried during the connection lifecycle to grab the `TLSVesion` used on the `SSLConnection`.
+    public var tlsVersionForHandler: TLSVersion {
+        return self.connection.getTLSVersionForConnection()
+    }
+}
+
+extension Channel {
+    ///  API to add the `NIOSSLHandler` to a channel to extract the `TLSVersion` during
+    ///  lifecycle events such as `whenSuccess` or `whenComplete`.
+    public func nioSSL_tlsVersionOnChannel() -> EventLoopFuture<NIOSSLHandler> {
```

Comment:
This returns the handler, not the version. It should probably return the version!